components/bank/landing.js

```diff
@@ -26,6 +27,13 @@ export default function Landing({ showButton, eventsCount }) {
               paddingTop: '96px'
             }}
           >
+            <Announcement
+              copy="Hack Club Bank: the ultimate financial tool for FIRST teams."
```

Comment:
what about 
copy: "Running a FIRST team? Learn how HC Bank works for you"
caption: "$250 grant, waived fees, and more!"

not super in love with the way i phrased it but i feel like we could be a little more forthcoming with the benefits? what do you think?

Reply:
The "Running a FIRST team" is a good idea– prefix the announcement with the audience so they know to look at it.

Another idea for title but more ridiculous:
"FRC, FTC, FLL, oh my!"
"Run on HCB this build season for a $250 grant, waived fees, and more!"

Reply:
Is HCB the official banking platform or just "the ultimate" because saying something is official sounds much more legit.

Reply:
> Is HCB the official banking platform or just "the ultimate" because saying something is official sounds much more legit.

i don't believe we're the official but we are working with FIRST to promote Bank amongst teams (i might be wrong though!)

Reply:
> ![image](https://user-images.githubusercontent.com/19589006/208322381-db662ac0-d3be-4757-966a-3af60ae48a32.png)
> 
> I have found a bug

ack! thanks for flagging it! change pushed :)

Reply:
![image](https://user-images.githubusercontent.com/19589006/208324030-ae03921d-7559-4b19-a653-6df55b58fc42.png)
 
it is still behind the nav